shared/pkg/scanner/grype/local_grype.go

```diff
@@ -62,7 +63,11 @@ func (s *LocalScanner) run(sourceType utils.SourceType, userInput string) {
 		ValidateByHashOnGet: false,
 	}
 	s.logger.Infof("Loading DB. update=%v", s.config.UpdateDB)
-	provider, metadataProvider, dbStatus, err := grype.LoadVulnerabilityDB(dbConfig, s.config.UpdateDB)
+
+	store, dbStatus, dbCloser, err := grype.LoadVulnerabilityDB(dbConfig, s.config.UpdateDB)
+	if dbCloser != nil {
+		defer dbCloser.Close()
```

Comment:
are we sure we want it? will it not delete the DB from filesystem?

Reply:
Not needed, I'm going to remove